lib/Template/Provider.pm

```diff
@@ -577,7 +577,7 @@ sub _load_compiled {
     # one needs to care about slash direction is when dealing
     # with Module::Name -> Module/Name.pm translation.
     my $fpath = File::Spec->rel2abs( $file );
-    $fpath =~ /^(.*)$/s;
+    $fpath =~ tr///; # untaint
     $fpath = $1;
```

Comment:
I'm not sure it works but you would at least have to remove this line.

Reply:
OK, I'll close this and just leave it to someone who knows what they're doing (i.e. not me) to fix if they care to.